test/unit/schema/postgres.js

```diff
@@ -5,10 +5,18 @@
 var tableSql;
 
 var PG_Client = require('../../../lib/dialects/postgres');
-var client    = new PG_Client({})
+var client    = new PG_Client({});
 
 var equal  = require('assert').equal;
 
+describe("PostgreSQL Config", function(){
+  it('check version', function(){
+    var version = 'v7.2.1-0';
```

Comment:
this should be e.g `8.5` and you can verify that it works by checking what kind of query is generated for request:
 
```
knex.schema.table(t => { 
  t.json('colName');
}).toSQL();
```